app/models/user.rb

```diff
@@ -6,5 +6,5 @@ class User < ApplicationRecord
   has_many :categories, dependent: :destroy
   validates :name, presence: true
   devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :validatable, :confirmable
+         :recoverable, :rememberable, :validatable
 end
```

Comment:
- [ ] Great job including tests for the components in your project :wink: 
- [ ] When I ran the tests, there was on test failing. So kindly work on that as well :wink: 

- Test Failed
![image](https://github.com/danielochuba/budget-app/assets/102699958/4c0231c9-a1f6-4bbd-8b67-cfac15290ca9)